README.md

```diff
@@ -27,6 +27,25 @@ The following image object is expected by the component -
 }
 ```
 
+The following image object is expect by the component (when used as a *node package to create a React component*) -
+
+```javascript
+{
+	url: 'someurl', //This will be passed into the image service
+	alt: 'your alt text',
+	picClass: 'any-bespoke-classes you-want-to-add', //Optional added to <picture> element along with n-image
+	imgClass: 'any-bespoke-classes you-want-to-add', //Optional added to <img> elements along with n-image__img
+	srcset: {
+		fallback: 500, //Optional pixel width for fallback image.
+		default: 200, //The default image size. This will be shown if non of the breakpoint sizes are matched or some haven't been specified
+		s: 300,
+		m: 500,
+		l: 700,
+		xl: 1000
+	}
+}
```

Comment:
is this right? don't you pass in parameters like attributes? an example of usage would be helpful